docs/rpc/deprecated/getStakeActivation.mdx

```diff
@@ -11,7 +11,7 @@ Returns epoch activation information for a stake account
 <Callout type={"warning"} title={"Deprecated Method"}>
   This method is expected to be removed in `solana-core` v2.0. Please use [this
   alternative
-  approach](https://gist.github.com/joncinque/ccd5166a857d629f30b2c01cd010041d)
+  approach](https://github.com/solana-developers/solana-rpc-get-stake-activation)
```

Comment:
the gist link is the recommendation from agave guide 
https://github.com/anza-xyz/agave/wiki/Agave-v2.0-Transition-Guide#step-2-review-removed-rpc-endpoints-and-sdk-elements

Reply:
I think since Jon C authored that repo, and it gives a rust and js example, we should linked to that.
We could also ask Rex to update the transition guide to use this repo that Jon created since it is more complete

Reply:
He moved it to solana-developers